src/System.IO.MemoryMappedFiles/src/Microsoft/Win32/SafeMemoryMappedViewHandle.cs

```diff
@@ -8,9 +8,7 @@
 
 namespace Microsoft.Win32.SafeHandles
 {
-#pragma warning disable 0618    // SafeBuffer is obsolete
     public sealed partial class SafeMemoryMappedViewHandle : SafeBuffer
-#pragma warning restore
```

Comment:
This is fine, but I'm curious... is it related to the test in some way?

Reply:
It is not - I should have made separate commit for it. It is related to discussion in: https://github.com/dotnet/corefx/issues/33279#issuecomment-440897888